system_tests/datastore.py

```diff
@@ -271,7 +287,13 @@ def test_projection_query(self):
         self.assertEqual(catelyn_stark_dict,
                          {'name': 'Catelyn', 'family': 'Stark'})
 
-        catelyn_tully_entity = entities[3]
+        if EMULATOR_DATASET is None:
```

Comment:
Yes, you can pass a `--data-dir` flag to tell it where to look. It will look for an index.yaml inside of the WEB-INF folder inside your `--data-dir`.

However, you should note that with `gcloud beta emulators` (vs gcd.sh) I don't think it is possible to tell the emulator to avoid building new indexes and fail instead.

Also huge :+1: to getting these working with the emulator!